sites/header/index.json-ld

```diff
@@ -30,5 +31,443 @@
+
+				"from_title": "Version 3.0 (with Auth)",
+				"from_code": "&lt;header&gt;\n\t&lt;div id=&quot;wb-bnr&quot; class=&quot;container&quot;&gt;\n\t\t&lt;div class=&quot;row&quot;&gt;\n\t\t\t&lt;!-- Language toggle [version 1.0] --&gt;\n\t\t\t&lt;!-- Branding [version 1.0] --&gt;\n\t\t\t&lt;!-- Search [version 1.0] --&gt;\n\t\t&lt;/div&gt;\n\t&lt;/div&gt;\n\t&lt;hr&gt;\n\t&lt;div class=&quot;container&quot;&gt;\n\t\t&lt;div class=&quot;row&quot;&gt;\n\t\t\t&lt;div class=&quot;col-md-8&quot;&gt;\n\t\t\t\t&lt;!-- Site Menu [version 2.0] --&gt;\n\t\t\t&lt;/div&gt;\n\t\t\t&lt;div class=&quot;col-xs-<del>5</del> col-xs-offset-<del>7</del> col-md-offset-0 col-md-4&quot;&gt;\n\t\t\t\t&lt;!-- Optional Authentication Section [version 1.0] --&gt;\n\t\t\t&lt;/div&gt;\n\t\t&lt;/div&gt;\n\t&lt;/div&gt;\n\t&lt;!-- Breadcrumbs [version 1.0] --&gt;\n&lt;/header&gt;"
+			},
+			"introduction": {
+				"en": "The difference with version 3.0 is the value for <code>col-xs-5</code> and <code>col-xs-offset-7</code> have been adjusted to a \"6\" so the button text don't brake and stays on one line for every viewports from extra-extra-small to extra-large (xxs, xxs, sm, md, lg and xl).",
+				"fr": "text... or HTML"
+			},
+			"instruction": {
+				"en": [
+					"This version requires the implementation of the <a href=\"../gcweb-menu/gcweb-menu-docs-en.html\">gcweb-menu version 2.0</a> where the <code>&lt;div class=\"container\"&gt;</code> has been relocated in the header so outside the \"gcweb-menu\" component."
+				],
+				"fr": []
+			}
+
+		}
+
+	],
+
+	"changeset": [
+		{
+			"@id": "_:cs_default",
+			"name": {
+				"en": "Version 4",
+				"fr": "Version 4"
+			},
+			"status": "stable",
+			"baseOnIteration": "_:iteration_4",
+
+			"detectableBy": "Text on how to detect",
+
+			"layout": [
+				"Row - Skip link first row centered",
+				"Row, cell - Language toggle top right",
+				"Cell - Logo brand top left",
+				"Cell - Search top right bellow language toggle",
+				"Row - Horizontal line bellow the logo brand and search box",
+				"Row, cell - Site top menu left bellow the branding",
+				"Cell - Authentication beside the top menu on its right side",
+				"Row - Breadcrumb under the top menu"
+			],
+			"style": [
+				"May have some specific style, need to be researched and centralized",
+				"Bootstrap 3 grid CSS: container, row, col-md-8, col-xs-6 col-xs-offset-6 col-md-offset-0 col-md-4 col-xs-12"
+			],
+			"semantic": {
+				"@type": "source-code",
+				"description": "Include with logic in liquid that support all variations",
+				"code": {
+					"@type": [ "@id", "rdf:HTML" ],
+					"@value": "includes/header.html"
+				}
+			},
+			"logic": "Not applicable",
+			"behaviour": "Not applicable",
+			"recognition": "The layout (disposition) of the component and the brand logo must be at top left.",
```

Comment:
Let's use another wording, the intent is to evaluate the change if it is perceivable from the user perceptiveness. 

Probably this item do belong more to the iteration object rather than the change set. The only way to evaluate if the change is perceptible for an user is after that change is made and a comparaison could occur.